dcp_prototype/backend/browser/api/app.py

```diff
@@ -0,0 +1,151 @@
+import os
+import sys
+
+import chalice
+from chalice import Chalice
+
+pkg_root = os.path.abspath(os.path.join(os.path.dirname(__file__), "chalicelib"))  # noqa
+sys.path.insert(0, pkg_root)  # noqa
+
+from browser.rds.browser_orm import (DBSessionMaker,
+                                     Project, File,
+                                     LibraryPrepProtocol, Tissue, Species,
+                                     LibraryPrepProtocolJoinProject, TissueJoinProject,
+                                     SpeciesJoinProject)
+
+app = Chalice(app_name='browser-api')
+
+
+@app.route('/')
+def index():
+    return "render documentation here"
+
+
+@app.route('/projects')
+def get_projects():
+    session = DBSessionMaker().session()
```

Comment:
can we move session to the global namespace? This way we can maintain the session if the lambda functions are warm and containers are re-used.

Reply:
Oh awesome. Good call